src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Remove.txt

```diff
@@ -65,4 +129,26 @@ Errors: Error 14-16: If provided, last argument must be 'All'. Is there a typo?|
 Errors: Error 35-39: If provided, last argument must be 'All'. Is there a typo?|Error 0-40: The function 'Remove' has some invalid arguments.
 
 >> Remove(t1, "All");
-Errors: Error 11-16: Invalid argument type (Text). Expecting a Record value instead.|Error 11-16: Cannot use a non-record value in this context.|Error 0-17: The function 'Remove' has some invalid arguments.
\ No newline at end of file
+Errors: Error 11-16: Invalid argument type (Text). Expecting a Record value instead.|Error 11-16: Cannot use a non-record value in this context.|Error 0-17: The function 'Remove' has some invalid arguments.
+
+>> Remove(Table(
+  {Planet:"Earth",Properties:{Color:"blue",Size:"small",Weight:1000,Moon:{Name:"Moon", Color:"Silver"}}},
+  {Planet:"Mars",Properties:{Color:"red",Size:"small",Weight:1000,Moon:{Name:"Daimon", Color:"Silver"}}},
+  {Planet:"Saturn",Properties:{Color:"red-brown",Size:"huge",Weight:99999,Moon:{Name:"Phobos", Color:"Yellow"}}}),
+  {Properties:{Moon:{Color:"Silver"}}})
+#Skip
```

Comment:
why are we adding with #skip?  We should at least include comment.

Reply:
These are test cases that I was planning to visit later. They were failing since I'm trying to mutate an immutable object. I saved them to a local file.